### PR TITLE
better nate than lever

### PR DESCRIPTION
**Pull request by xyzrus**
review when hippossible thnx

### CODE REVIEW

File: _posts/randy/2022-09-10-tech-juggling-reality [hunk 1]
```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title: "New Tech Success - Juggling Cycle"
+author: randy
+permalink: /@randy/draft
```
Comment (adamjberg) [inline _posts/randy/2022-09-10-tech-juggling-reality]: Make a real link

File: _posts/randy/2022-09-10-tech-juggling-reality [hunk 2]
```diff
@@ -0,0 +1,171 @@
+---
```
Comment (adamjberg) [inline _posts/randy/2022-09-10-tech-juggling-reality]: File needs .md extension

File: _posts/randy/2022-09-10-tech-juggling-reality [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+[3 Ways to Prepare Your Business for the Inevitable Quantum Revolution – Entrepreneur](https://www.entrepreneur.com/article/433374)
+
+[Business Technology Trends - Rasmussen U](https://www.rasmussen.edu/degrees/business/blog/business-technology-trends)
+
+[Create a Tech Startup in 12 Steps](https://onlinedegrees.sandiego.edu/how-to-create-a-tech-startup-company-in-12-steps-2)
+
+[Launching Tech Startups - Complete Guide](https://www.neoito.com/blog/launching-tech-startup-complete-guide)
+
+[Industrial Digitalization - Hitachi](https://social-innovation.hitachi/en-us/think-ahead/manufacturing/industrial-digitalization-for-smart-manufacturing) {NOTE – extensive use of buzz-phrases}
```
Comment (adamjberg) [inline _posts/randy/2022-09-10-tech-juggling-reality]: I can't comment on the wav file below.... but are you hoping to include in the article?  I can show you the syntax for that, but we should include as mp3 not wav.

**Review by adamjberg — COMMENTED** [review]
I like how this one flowed.  Something rhythmic about the writing, which also kind of ties in well with the clown juggling.  Gave me some things to think about.

Some comments/questions.

File: _posts/randy/2022-09-10-tech-juggling-reality (hunk 2 above)
Comment (xyzrus) [reply _posts/randy/2022-09-10-tech-juggling-reality]: du.md edum

File: _posts/randy/2022-09-10-tech-juggling-reality (hunk 1 above)
Comment (xyzrus) [reply _posts/randy/2022-09-10-tech-juggling-reality]: get real

File: _posts/randy/2022-09-10-tech-juggling-reality (hunk 3 above)
Comment (xyzrus) [reply _posts/randy/2022-09-10-tech-juggling-reality]: i could not find the devpage where you had an audio file to compare code - will find a way to convert to mp3 and hold my nose same time. I wish to put the audio clip below the video - will add one to each article so far (some day).

Comment (adamjberg) [reply _posts/randy/2022-09-10-tech-juggling-reality]: I used to have an example on one of my articles.... but ended up changing to a Kaizen embed.  Which reminds me, that that's what we should plan to do for yours as well.

Until then, you can use:

```
<audio controls="controls">
  <source type="audio/mp3" src="{{"/assets/audio/impossible-quote.mp3" | relative_url}}"></source>
  <p>Your browser does not support the audio element.</p>
</audio>
```

Just make sure to replace `impossible-quote.mp3` with the name of your file and put it in the `audio` folder under the assets folder.

**Review by adamjberg — CHANGES_REQUESTED** [review]
Added details for how to add mp3.  You also marked my comments as resolved but I don't see them resolved

**Review by adamjberg — APPROVED** [review]

**Pull request merged**